static/app/components/issues/groupList.tsx

```diff
@@ -35,8 +35,11 @@ const defaultProps = {
   useFilteredStats: true,
   useTintRow: true,
   narrowGroups: false,
+  withColumns: ['graph', 'event', 'users', 'assignee', 'lastTriggered'] as Column[],
```

Comment:
Should be `satisfies Column[]` ?

Reply:
Nice catch! Wasn't aware of `satisfies`, seems to solve the problem of `as Column[]`, because with the `as` keyword, you get type hinting, but you can still put arbitrary values in the array.
Will update that

Reply:
updated